main.js

```diff
@@ -0,0 +1,44 @@
+const mobileMenu = document.querySelector('.mobile-header');
```

Comment:
**Kindly work on the requested changes:**
Provide the requested design by the Figma Template.
_Your `Mobile Menu` is missing to fit the 100% of the body._

**_Yours:_**

![Screen Shot 2021-12-14 at 0 06 32](https://user-images.githubusercontent.com/22258274/145942656-57f6a91b-29c3-464c-b26c-3353980167cf.png)

#

**Figma Template:**
![Screen Shot 2021-12-14 at 0 08 49](https://user-images.githubusercontent.com/22258274/145942705-a2d0d31b-5ad5-411f-9f9f-bbf78d90ba11.png)